seqr/utils/elasticsearch/constants.py

```diff
@@ -108,6 +108,7 @@
     'AN': {},
     'Hom': {},
     'Hemi': {},
+    'ID': {'format_value': str, 'default_value': None},
```

Comment:
There is a field `sv_callset_het` for genome SVs which is still not been loaded into the search response. Should I add one more field to `POPULATION_FIELD_CONFIGS`?